src/components/Cards/MinutesItemCard/MinutesItemCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
```

Comment:
Is this needed? I thought it was imported at a higher level but :shrug:

Reply:
Got it. Commenting here to confirm that I removed this line.

Reply:
Haha sorry to potentially switch things up, but since we actually import Mozilla protocol in every other component that uses it we may want to keep it in just to be consistent?

With that said, if we are globally importing this library and it would work to just remove all the component-specific imports we may want to open an issue where we remove all the imports elsewhere. @tohuynh do you have any thoughts?